File: CS/WindowsFormsApplication32/MyTabbedMdiManager/MySkinViewInfoRegistrator.cs
using DevExpress.XtraTab;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.ViewInfo;

namespace CustomTabbedMdiManager
{
	public class MySkinViewInfoRegistrator : SkinViewInfoRegistrator
	{
		public MySkinViewInfoRegistrator()
		{
		}

		public override BaseTabHeaderViewInfo CreateHeaderViewInfo(BaseTabControlViewInfo viewInfo)
		{
			return new MySkinTabHeaderViewInfo(viewInfo);
		}

		public override BaseTabControlViewInfo CreateViewInfo(IXtraTab tabControl)
		{
			return new MySkinTabControlViewInfo(tabControl);
		}
	}
}

File: VB/WindowsFormsApplication32/MyTabbedMdiManager/MySkinViewInfoRegistrator.vb
Imports DevExpress.XtraTab
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.ViewInfo

Namespace CustomTabbedMdiManager

    Public Class MySkinViewInfoRegistrator
        Inherits SkinViewInfoRegistrator

        Public Sub New()
        End Sub

        Public Overrides Function CreateHeaderViewInfo(ByVal viewInfo As BaseTabControlViewInfo) As BaseTabHeaderViewInfo
            Return New MySkinTabHeaderViewInfo(viewInfo)
        End Function

        Public Overrides Function CreateViewInfo(ByVal tabControl As IXtraTab) As BaseTabControlViewInfo
            Return New MySkinTabControlViewInfo(tabControl)
        End Function
    End Class
End Namespace
